src/utilities/string.test.ts

```diff
@@ -0,0 +1,50 @@
+import { toTitle, UrlSafeBase64 } from './string';
+
+describe('toTitle()', () => {
+  test('upper-case the first non-blank character normal usage', () => {
+    expect(toTitle('normal title'));
```

Comment:
expect().toEqual()?

Reply:
My bad. Forgot to complete the unit tests of `toTitle`